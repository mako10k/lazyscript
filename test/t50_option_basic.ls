!{
  ~M <- ((~prelude requirePure) "lib/Option.ls");
  ~~println (~to_str (((~M .Option) Some) 1));
  ~~println (~to_str (((~M .map) (\~x -> ~add ~x 1)) (((~M .Option) Some) 2)));
  ~~println (~to_str ((((~M .getOrElse) 99) (((~M .Option) None)))));
};
