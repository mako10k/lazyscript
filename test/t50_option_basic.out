Some 1
Some 3
99
()
